data/human/jobs.txt

```diff
@@ -2502,6 +2502,46 @@ mission "Released Prisoners [2]"
 		payment 3000 150
 		dialog "The released prisoners walk into the spaceport with nothing but discharge paperwork and the clothes on their backs. You collect your payment of <payment>."
 
+mission "Passenger Runoff [0]"
```

Comment:
Maybe have an `on accept` dialog, like some of the Southern suspicious missions?